src/Catty/PlayerEngine/DataModel/CBSpriteNode/CBSpriteNode.swift

```diff
@@ -237,4 +239,11 @@ class CBSpriteNode: SKSpriteNode {
         return true
     }
 
+    @objc func isFlipped() -> Bool {
```

Comment:
Please test this method in `CBSpriteNodeTests`.

Reply:
Done!